app/serializers/category_serializer.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class CategorySerializer
+  include JSONAPI::Serializer
+
+  set_type :categories
+  set_id :key
```

Comment:
@martyn-w @merlinc Just been testing this locally and realised that this should really be returning the UUID `id` column as `id` so that it matches with the relationships in the rest of the JSONAPI payload. `key` will instead move to an attribute. I'll update the serializer and spec accordingly.